evalml/tests/automl_tests/test_automl_search_classification.py

```diff
@@ -1495,6 +1495,28 @@ def test_automl_search_sampler_k_neighbors_param(sampler, has_minimal_dependenci
         seen_under = False
         for comp in pipeline.component_graph:
             if comp.name == sampler:
-                assert comp.parameters["k_neighbors"] == 2
+                assert comp.parameters["k_neighbors_default"] == 2
                 seen_under = True
         assert seen_under
+
+
+@pytest.mark.parametrize(
+    "parameters", [None, {"SMOTENC Oversampler": {"k_neighbors_default": 5}}]
+)
+def test_automl_search_sampler_k_neighbors_no_error(
+    parameters, has_minimal_dependencies, fraud_100
+):
+    # automatically uses SMOTE
+    if has_minimal_dependencies:
+        pytest.skip("Skipping tests since imblearn isn't installed")
+    # split this from the undersampler since the dictionaries are formatted differently
```

Comment:
are these docstrings correct? I'm confused what they refer to 😕

Reply:
Ah, mistake from copying!